linkml_runtime/utils/schemaview.py

```diff
@@ -820,41 +820,37 @@ def enum_parents(
     @lru_cache(None)
     def permissible_value_parent(
         self, permissible_value: str, enum_name: ENUM_NAME
-    ) -> list[str | PermissibleValueText] | None:
+    ) -> list[str | PermissibleValueText]:
         """:param enum_name: child enum name
         :param permissible_value: permissible value
         :return: all direct parent enum names (is_a)
         """
         enum = self.get_enum(enum_name, strict=True)
-        if enum:
```

Comment:
`get_enum(enum_name, strict=True)` throws an error if `enum` is not found, so checking for `enum` is unnecessary